src/govuk/components/summary-list/_index.scss

```diff
@@ -87,27 +89,54 @@
   }
 
   .govuk-summary-list__actions-list {
+    display: flex;
```

Comment:
I'm wondering if we can avoid the flex change by removing the errant whitespace in the template instead, using the [minus symbol to strip out whitespace](https://mozilla.github.io/nunjucks/templating.html#whitespace-control) within the list?

E.g:

```
{%- for action in row.actions.items -%}
    <li class="govuk-summary-list__actions-list-item">{{ _actionLink(action) | indent(18) | trim }}</li>
{%- endfor -%}
```

Reply:
Regardless, I think it'd be good for the whitespace fix to be pulled out as a separate commit on this PR as it's not necessary to fix the alignment issue.